src/WebJobs.Script/Extensions/ActionExtensions.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.Azure.WebJobs.Script
 {
     public static class ActionExtensions
     {
-        public static Action<T> Debounce<T>(this Action<T> func, int milliseconds = 300)
+        public static Action<T> Debounce<T>(this Action<T> func, CancellationToken cancellationToken, int milliseconds = 300)
```

Comment:
Can we switch this to a default argument setting the `cancellationToken` to `CancellationToken.None` to avoid impacting all of the existing consumers. That will significantly reduce the number of files changed as a result.